server/src/main/kotlin/suwayomi/tachidesk/manga/impl/Page.kt

```diff
@@ -49,7 +50,11 @@ object Page {
         val chapterId = chapterEntry[ChapterTable.id].value
 
         val pageEntry =
-            transaction { PageTable.select { (PageTable.chapter eq chapterId) and (PageTable.index eq index) }.first() }
+            transaction {
+                PageTable.select { (PageTable.chapter eq chapterId) }
+                    .orderBy(PageTable.index to SortOrder.ASC)
+                    .limit(1, index.toLong()).first()
+            }
```

Comment:
Some extensions do not start the index from 0. This causes the viewer and the API  to break. What I have done here is get the pages sorted by index, and then take the row at the index needed. This ensures that we never fetch a page out of order and also does not rely on the hard value of index for which tachiyomi-extension does not have any standards

Reply:
Done. https://github.com/Suwayomi/Tachidesk-Server/pull/491/files

Reply:
Remove the change from this PR.

Reply:
Done